src/load/azext_load/tests/latest/resources/config.yaml

```diff
@@ -9,6 +9,9 @@ zipArtifacts:
 failureCriteria:
   - avg(requests_per_sec) > 78
   - percentage(error) > 50
+  - p75(response_time_ms) > 380
+  - p99(response_time_ms) > 520
+  - p99.9(response_time_ms) > 540
```

Comment:
add percentile for p99.99 etc.

Reply:
we don't need to add test cases for all, just needed to verify that adding new percentiles in service will work for cli as well directly